pkg/providers/imagefamily/azlinux.go

```diff
@@ -60,6 +62,7 @@ func (u AzureLinux) DefaultImages() []DefaultImageOutput {
 				scheduling.NewRequirement(v1.LabelArchStable, v1.NodeSelectorOpIn, corev1beta1.ArchitectureAmd64),
 				scheduling.NewRequirement(v1alpha2.LabelSKUHyperVGeneration, v1.NodeSelectorOpIn, v1alpha2.HyperVGenerationV1),
 			),
+			Distro: "aks-azurelinux-v2",
```

Comment:
Can we make these consts, and can we import them from agentbaker?

Reply:
Will consider that along with potential delegation of this field to the provision server.